test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinExporterTests.cs

```diff
@@ -89,6 +90,52 @@ public void ZipkinExporter_BadArgs()
             Assert.Throws<ArgumentNullException>(() => builder.AddZipkinExporter());
         }
 
+        [Fact]
+        public void ZipkinExporter_SuppresssesInstrumentation()
+        {
+            const string ActivitySourceName = "zipkin.test";
+            Guid requestId = Guid.NewGuid();
+            TestActivityProcessor testActivityProcessor = new TestActivityProcessor();
+
+            int endCalledCount = 0;
+
+            testActivityProcessor.EndAction =
+                (a) =>
+                {
+                    endCalledCount++;
+                };
+
+            var exporterOptions = new ZipkinExporterOptions();
+            exporterOptions.ServiceName = "test-zipkin";
+            exporterOptions.Endpoint = new Uri($"http://{this.testServerHost}:{this.testServerPort}/api/v2/spans?requestId={requestId}");
+            var zipkinExporter = new ZipkinExporter(exporterOptions);
+            var exportActivityProcessor = new BatchExportActivityProcessor(zipkinExporter, scheduledDelayMilliseconds: 1);
```

Comment:
We don't need to have such a low latency as it is going to waste CPU.
We can either call processor `ForceFlush` or `Shutdown`.

Reply:
Good call, hadn't thought about `ForceFlush`. I'd still rather use the `SimpleExportActivityProcessor`, but will play with that after this PR. Something about both the `Thread.Sleep` and `ForceFlush` approach seems a little non-deterministic, I'm concerned this may lead to flickers or false positives.

Reply:
ForceFlush is a blocking API that has deterministic behavior https://github.com/open-telemetry/opentelemetry-dotnet/blob/552728a7bcdf62930b252b24c48ef614ecbfefb3/src/OpenTelemetry/Trace/ActivityProcessor.cs#L78.